draft-meynell-panrg-scion-research-questions.md

```diff
@@ -471,11 +473,18 @@ Is reverse path refresh a relevant problem?
 -->
 
 # Interfaces for Path Awareness
+The questions posed here are:
 
-* IPv6 in the Data Plane
-* SCION-IP translation
+* Are there any existing protocols/solutions that can carry path aware traffic?
+* If there are not, what would it require?
```

Comment:
In the next section you show that there are existing protocols/solutions, so the question is already answered?
Remove this?

Reply:
I think there aren't. The list is there because ISPs will mention IPv6, and vendors will say that it is not really a solution. Same for the scion-ip translation approach.
I've modified the sentence to `If there are not suitable out-of-the-box, what would it require to make them work?`

Reply:
Okay, so I think I misunderstood what this is about. Maybe we should be more clear what this question is about?

> * Are there any existing protocols/solutions that could be adapted to carry SCION traffic?
>  * If there are not suitable out-of-the-box, what would be required to make them work?

I think the question isn't really whether something could be adapted, but whether it would be beneficial if something would be adapted.

I think the question may be more like: 
```
* What existing protocols/solutions do we need to be interoperable/compatible with SCION?
* If they are not interoperable/compatible with SCION yet, how can this be achieved?
```
Does that make sense?

Reply:
The question is about how to make SCION interoperable with existing tech. such as IPv6. And, if not out-of-the-box, what is the effort required for that.
It's more about reusing something that exists to carry SCION traffic, that enabling SCION for existing things. Essentially: what is the least resistance path to allow ISPs to carry SCION traffic?
I like your second sentence better than my original, I propose something like:
```
* Are there any existing protocols/solutions that could carry SCION traffic?
* If they are not interoperable/compatible with SCION yet, how can this be achieved?
```
What do you think?